partner_contact_personal_info/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Contact Personal Information",
```

Comment:
Maybe a name like Partner Contact Weight and height could be more accurate, as there are several modules that increase personal info like date of birth...